src/js/utils/Template.js

```diff
@@ -3,6 +3,18 @@
   var templates = {};
 
   ns.Template = {
+    /**
+     * Takes in a templateId and values to replace the keys found in the template
+     * Requires Node-Webkit environment for file system access
+     * @param {string} templateId - ID used to retrieve the correct template from misc-templates.html
+     * @param {string} templateValues - The values to fill in the template with.
```

Comment:
`templateValues` is not a string, right?
```suggestion
     * @param {Object} templateValues - The values to fill in the template with.
```